apps/landingPage/package-lock.json

```diff
@@ -0,0 +1,15743 @@
+{
```

Comment:
could we stick to having `yarn.lock` only please. No package-lock.json 😊

Reply:
Ok! never again